README.md

```diff
@@ -25,22 +25,23 @@ It's a **Spotlight** plugin to display informations of unsupported images (**Web
 
 ### Installation
 
-2 choices : 
+3 choices : 
 
+- Via [Homebrew Cask](https://brew.sh/): `brew cask install qlimagesize`
+- Download manually the latest build from https://github.com/Nyx0uf/qlImageSize/releases/tag/1.6.1 and save it to the `~/Library/QuickLook` folder
```

Comment:
IMO should just point to the release page https://github.com/Nyx0uf/qlImageSize/releases

Reply:
What about https://github.com/Nyx0uf/qlImageSize/releases/latest?